README.md

```diff
@@ -26,7 +26,7 @@ import { TLNetwork } from 'trustlines-clientlib'
 
 Use the following configuration to connect to the currently deployed test setup.
 
-**NOTE: The [trustlines-network contracts](https://github.com/trustlines-network/contracts) are deployed on the Kovan testnet. Some Kovan Test ETH is therefore required to interact with the contracts in this setup.**
+**NOTE: The [trustlines-network contracts](https://github.com/trustlines-network/contracts) are deployed on the (_Trustlines_) Laika testnet. Some Laika test `TLC` is therefore required to interact with the contracts in this setup.**
```

Comment:
Lets add a link to the Laika blockexplorer. The sentence about the tlc is still right.

Reply:
Good idea. I have done so. :heavy_check_mark: